src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -278,7 +278,8 @@ internal IOperation CreateInternal(BoundNode boundNode)
                     return CreateRangeExpressionOperation((BoundRangeExpression)boundNode);
                 case BoundKind.SwitchSection:
                     return CreateBoundSwitchSectionOperation((BoundSwitchSection)boundNode);
-                case BoundKind.SwitchExpression:
+                case BoundKind.UnconvertedSwitchExpression:
```

Comment:
>UnconvertedSwitchExpression [](start = 31, length = 27)

can we get an unconverted switch here?

Reply:
Yes, in error scenarios.

---
In reply to: [301822553](https://github.com/dotnet/roslyn/pull/37052#discussion_r301822553) [](ancestors = 301822553)